src/scss/components/optional/_order-card.scss

```diff
@@ -74,6 +82,10 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
             width: 100%;
         }
 
+        .c-yourFavourites {
```

Comment:
Same here – I would think that you need a `c-orderCard-something` type class to hook onto rather than using `c-yourFavourites`, as  otherwise you'll be mixing component classes (which you should try not to do)

Reply:
Will do!